packages/slipshow/slipshow.0.5.0/opam

```diff
@@ -0,0 +1,64 @@
+opam-version: "2.0"
+synopsis: "A compiler from markdown to slipshow"
+description:
+  "Slipshow is an engine to write slips, a concept evolved from slides."
+maintainer: ["Paul-Elliot"]
+authors: ["Paul-Elliot"]
+license: ["GPL-3.0-or-later" "ISC" "BSD-3-Clause" "Apache-2.0" "OFL-1.1"]
+tags: ["slipshow" "presentation" "slideshow" "beamer"]
+homepage: "https://github.com/panglesd/slipshow"
+doc: "https://slipshow.readthedocs.io"
+bug-reports: "https://github.com/panglesd/slipshow/issues"
+depends: [
+  "ocaml" {>= "4.14"}
+  "dune" {>= "3.6"}
```

Comment:
```suggestion
  "dune" {>= "3.17"}
```
https://opam.ci.ocaml.org/github/ocaml/opam-repository/commit/afffbb86d2d925147de2c7283df336addbc3ff31/variant/compilers,5.3,slipshow.0.5.0,lower-bounds
as `downgrade dune 3.19.1 to 3.15.3`
is failing with `Error: Version 3.17 of the dune language is not supported`